cluster/examples/kubernetes/ceph/csi/cephfs/snapshotclass.yaml

```diff
@@ -1,5 +1,4 @@
 ---
-# apiVersion: snapshot.storage.k8s.io/v1alpha1
```

Comment:
can we not use cephfs with alpha snapshots with the latest rook?

Reply:
They can use alpha snapshots https://github.com/rook/rook/blob/master/Documentation/ceph-csi-snapshot.md#prerequisites
> Note: If only Alpha snapshots are available, enable snapshotter in rook-ceph-operator-config or helm chart values.yaml, change the external-snapshotter image to k8s.gcr.io/sig-storage/csi-snapshotter:v1.2.2 and refer to the alpha snapshots documentation

As per comment https://github.com/rook/rook/pull/7711#discussion_r626004824, I only mentioned v1beta1 and v1 in example yaml files. I missed to remove that comment  in cephfs snapshotclass.yaml alone by mistake :sweat: